stagecoach/jobs/publishers/evm-publisher/app.py

```diff
@@ -0,0 +1,118 @@
+import asyncio
+import json
+import os
+
+import boto3
+from dotenv import load_dotenv
+from empiric.core import SpotEntry
+from empiric.core.logger import get_stream_logger
+from empiric.core.mixins.evm import EvmHelper
+from empiric.core.utils import currency_pair_to_pair_id, felt_to_str
+from empiric.publisher.client import EmpiricPublisherClient
+from empiric.publisher.fetchers import BitstampFetcher, CexFetcher, GeminiFetcher
+
+load_dotenv()
+ASSETS = [
+    {"type": "SPOT", "pair": ("BTC", "USD"), "decimals": 8},
+    {"type": "SPOT", "pair": ("BTC", "EUR"), "decimals": 8},
+    {"type": "SPOT", "pair": ("ETH", "USD"), "decimals": 8},
+    {"type": "SPOT", "pair": ("DAI", "USD"), "decimals": 8},
+    {"type": "SPOT", "pair": ("USDT", "USD"), "decimals": 6},
+    {"type": "SPOT", "pair": ("USDC", "USD"), "decimals": 6},
+    {"type": "SPOT", "pair": ("BNB", "USD"), "decimals": 8},
+    {"type": "SPOT", "pair": ("MATIC", "USD"), "decimals": 8},
+    {"type": "SPOT", "pair": ("AAVE", "USD"), "decimals": 8},
+]
+PAIRS = [currency_pair_to_pair_id(*p["pair"]).encode() for p in ASSETS]
+PROVIDER_URI = os.environ["PROVIDER_URI"]
+CHAIN_ID = int(os.environ["CHAIN_ID"])
+ORACLE_ADDRESS = os.environ["ORACLE_ADDRESS"]
+SECRET_NAME = os.environ["SECRET_NAME"]
+REGION_NAME = os.environ["REGION_NAME"]
+PUBLISHER = os.environ["PUBLISHER"]
+
+logger = get_stream_logger()
+
+
+def handler(event, context):
+    entries_ = asyncio.run(_handler(ASSETS))
+    return {
+        "success": len(entries_),
+    }
+
+
+# def _get_pvt_key():
+#     secret_name = "zkSyncTestnetPrivateKey"
+#     region_name = "us-west-1"
+
+#     # Create a Secrets Manager client
+#     session = boto3.session.Session()
+#     client = session.client(service_name="secretsmanager", region_name=region_name)
+#     get_secret_value_response = client.get_secret_value(SecretId=secret_name)
+#     return (
+#         "0x"
+#         + json.loads(get_secret_value_response["SecretString"])[
+#             "ZKSYNC_PUBLISHER_PRIVATE_KEY"
+#         ]
+#     )
+
+
+def _get_private_and_public_keys(secret_name, region_name):
+    session = boto3.session.Session()
+    client = session.client(service_name="secretsmanager", region_name=region_name)
+    secret_value_response = client.get_secret_value(SecretId=secret_name)
+    keys = json.loads(secret_value_response["SecretString"])
+    return keys["PUBLISHER_PRIVATE_KEY"], keys["PUBLISHER_PUBLIC_KEY"]
+
+
+async def _handler(assets):
+    publisher_private_key, sender_address = _get_private_and_public_keys(
+        secret_name=SECRET_NAME, region_name=REGION_NAME
+    )
+    evm_publisher_client = EvmHelper(
```

Comment:
@rlkelly Previously this object was callsed `zksync_publisher_client`. I changed it to `evm_publisher_client`. If you want me to change the name to something else, please let me know.

Reply:
makes sense to me